opentelemetry/manifest.json

```diff
@@ -0,0 +1,36 @@
+{
+  "display_name": "OpenTelemetry",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "opentelemetry",
+  "metric_prefix": "otel.",
+  "metric_to_check": [
+    "otel.datadog_exporter.metrics.running",
+    "otel.datadog_exporter.traces.running"
+  ],
```

Comment:
The Datadog exporter on the OpenTelemetry Collector currently has two independent components: metrics and traces. We check for either metric to say that the Collector is running.